src/videojs-contrib-hls.js

```diff
@@ -374,7 +279,8 @@ class HlsHandler extends Component {
     if (typeof this.options_.bandwidth !== 'number') {
       // only use Android for mobile because iOS does not support MSE (and uses
       // native HLS)
-      this.options_.bandwidth = videojs.browser.IS_ANDROID ? 500000 : 4194304;
+      this.options_.bandwidth =
+        videojs.browser.IS_ANDROID ? BANDWIDTH_MOBILE : BANDWIDTH_BROADBAND;
```

Comment:
Might be easier read/more precise as `INITIAL_BANDWIDTH_MOBILE` and `INITIAL_BANDWIDTH_DESKTOP`